makelib/golang.mk

```diff
@@ -197,6 +202,11 @@ go.fmt: $(GOFMT)
 	@gofmt_out=$$($(GOFMT) -s -d -e $(GO_SUBDIRS) $(GO_INTEGRATION_TESTS_SUBDIRS) 2>&1) && [ -z "$${gofmt_out}" ] || (echo "$${gofmt_out}" 1>&2; $(FAIL))
 	@$(OK) go fmt
 
+go.fmt.simplify: $(GOFMT)
+	@$(INFO) gofmt simplify
+	@$(GOFMT) -l -s -w $(GO_SUBDIRS) || $(FAIL)
```

Comment:
what about `$(GO_INTEGRATION_TESTS_SUBDIRS)`?

Reply:
Good point, I'll add that.